SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -115,13 +128,54 @@ private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 								}
 								else
 								{
-									if (bldr.Length > 0)
-										bldr.AppendLine();
+									
+
+									bool endOfFile = false; 
+									bool currentPackage = false; 
+									bool printLine = false; 
 
 									var headerText = currentLine.Substring(levelHeader.Length);
 									if (!headerText.EndsWith(":"))
 										headerText += ":";
-									bldr.AppendLine(headerText);
+
+									//exclude unnecessary headers from changelog file
+									for (int i = _currentIndex + 1; i < _markdownLines.Length - 1; i++)
```

Comment:
On a quick look the `for` loop gives the impression that we process all lines to the end, but we only look at the next few lines. Maybe it would be clearer to put the loop in a method like `bool SkipHeader`?